addons/api/mirage/factories/host-catalog.js

```diff
@@ -52,8 +52,9 @@ export default factory.extend({
 
   worker_filter: function () {
     if (
-      this.type === TYPE_HOST_CATALOG_DYNAMIC &&
-      this.plugin?.name === TYPE_HOST_CATALOG_PLUGIN_AWS
+      (this.type === TYPE_HOST_CATALOG_DYNAMIC &&
+        this.plugin?.name === TYPE_HOST_CATALOG_PLUGIN_AWS) ||
+      this.plugin?.name === TYPE_HOST_CATALOG_PLUGIN_GCP
```

Comment:
Shouldn't the parentheses be around the names?

```suggestion
      this.type === TYPE_HOST_CATALOG_DYNAMIC &&
        (this.plugin?.name === TYPE_HOST_CATALOG_PLUGIN_AWS ||
      this.plugin?.name === TYPE_HOST_CATALOG_PLUGIN_GCP)
```